ocean-client/src/vault-maxi.ts

```diff
@@ -104,14 +109,20 @@ export async function main(event: maxiEvent,context: any): Promise<Object> {
 
         const oldRatio = +vault.collateralRatio
         const nextRatio = nextCollateralRatio(vault)
-        const usedCollateralRatio = Math.min(+vault.collateralRatio, nextRatio)
+        const usedCollateralRatio = BigNumber.min(vault.collateralRatio, nextRatio)
         console.log("starting with " + vault.collateralRatio + " (next: " + nextRatio + ") in vault, target "
             + settings.minCollateralRatio + " - " + settings.maxCollateralRatio + " token " + settings.LMToken)
         let exposureChanged = false
         //first check for decreaseExposure
         // if no decrease necessary: check for reinvest (as a reinvest would probably trigger an increase exposure, do reinvest first)
         // no reinvest -> check for increase exposure
```

Comment:
small: comment is not updated to reflect new changes regarding completely removing exposure